nisqa/NISQA_model.py

```diff
@@ -73,8 +73,11 @@ def predict(self):
                     
         if self.args['output_dir']:
             self.ds_val.df['model'] = self.args['name']
+            output_file_name = 'NISQA_results.csv'
+            if self.args['output_file_name']:
```

Comment:
Could you change this as follows to make sure it's backward compatible? For example, run_evaluate.py is also calling this function.

```suggestion
            if 'output_file_name' in self.args:
```